benchmarks/src/main/java/org/questdb/CoalesceBenchmark.java

```diff
@@ -81,7 +81,7 @@ public long getLong(int col) {
 
     @Benchmark
     public long testBaseline() {
-        Function function = new CoalesceFunctionFactory.LongCoalesceFunction(0, constFunctions, constFunctions.size());
+        Function function = new CoalesceFunctionFactory.LongCoalesceFunction(constFunctions, constFunctions.size());
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
Analysis of this package determined that this line of code might contain a resource leak. A resource leak can slow down or crash your system.

**Fix**
Consider closing the following resource in a try-finally block: *function*. Execution paths that do not contain closure statements were detected.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).